src/effects.js

```diff
@@ -24,14 +24,19 @@ const never =
 
 export class Effects /*::<a>*/ {
   static task /*::<a>*/(task/*:Task<Never, a>*/)/*:Effects<a>*/ {
-    return new Effects(task)
+    console.warn('Effects.task is deprecated please use Effects.perform instead')
```

Comment:
Do you actually have any users relying on this? I.e., is this a warning that anyone outside of browser.html could ever see? If not, let's just remove `task` and ensure it's not used in browser.html.